src/app/services/date-select.service.ts

```diff
@@ -1,5 +1,6 @@
 import { Injectable } from '@angular/core';
 import { Subject } from 'rxjs';
+import * as moment from 'moment';
```

Comment:
does
`import { Moment } from 'moment';`
not work? You would save yourself having to write `moment.Moment` every time.

Reply:
Not sure why I had left it that way at first, but it fine, so I'm switching it over.